mail-client/mutt-wizard/Manifest

```diff
@@ -0,0 +1 @@
+DIST mutt-wizard-0_pre20200506.tar.gz 32139 BLAKE2B 81fb86bce808db3e4f65788d2d3a59b9a7d18f210dc29e525773f596594bc2cba42424b73a50a6a92e5271c160f4d332943b8fa9526f3d44df5da8f96ffd1dc7 SHA512 26ff620b570f1fb6a9fed31d56233a2a67055c6a3352dd547fe7c258cdf2476e47fd87478d678fce953b921d0b6ce9ab3bbec563cbc616c83db1bc5a8047bd26
```

Comment:
Would you like to update this to latest available commit? I see a lot has happened in a month.

Reply:
Ah, yes thanks for that.

I've done the same.